awx/main/tests/unit/settings/test_k8s_resource_setttings.py

```diff
@@ -37,6 +37,8 @@ def test_SYSTEM_TASK_ABS_MEM_conversion(value, converted_value, mem_capacity):
         assert convert_mem_str_to_bytes(value) == converted_value
         assert get_corrected_memory(-1) == converted_value
         assert get_mem_effective_capacity(-1) == mem_capacity
```

Comment:
in this assertion, we will want to pass `is_control_node=True`, since it is a control node

same with get_cpu_effective_capacity

Reply:
Thank you so much!! I was going a little crazy trying to figure out how to get these two to pass.